mirrord/layer/src/socket/ops.rs

```diff
@@ -493,21 +526,24 @@ pub(super) fn accept(
             })?
     };
 
-    let (local_address, remote_address) = {
+    let (local_ip_address, remote_ip_address) = {
         CONNECTION_QUEUE
             .lock()?
             .pop_front(id)
             .bypass(Bypass::LocalFdNotFound(sockfd))
             .map(|socket| (socket.local_address, socket.remote_address))?
     };
 
+    let remote_address = SocketAddress::Ip(remote_ip_address);
```

Comment:
Weird naming, it's still an `ip_address` right? Is it possible to just override `remote_ip_address`?

Reply:
Or even do the conversion to SocketAddress::Ip in the map?

Reply:
Moved into map.